src/components/Order_page/Order_card/Order_card.scss

```diff
@@ -0,0 +1,45 @@
+.order_card{
+  background-color: #F8F9FA;
+  width: 100%;
+  padding: 20px;
+  position: fixed;
+  bottom: 0;
+  right: 0;
+
+}
+h3{
+  color: #007bff;
+  padding-bottom: 16px;
+  font-weight: 600;
+}
```

Comment:
Worth nesting this in the .order_card selector to make sure you don't clash with any other h3 tags